lib/ddtrace/opentracing.rb

```diff
@@ -0,0 +1,23 @@
+module Datadog
+  # Namespace for ddtrace OpenTracing implementation
+  module OpenTracing
```

Comment:
In both our python and go opentracing implementations we refer to the tracer and name the sub-module as 'opentracer'.

This is probably not a big deal, but we may wish to be consistent across our implementations.

Reply:
I see. For consistency with all other Datadog tracers, I'll change `OpenTracing` to `OpenTracer`.